Sources/GDSCommon/Patterns/GDSError/GDSErrorViewModel.swift

```diff
@@ -9,3 +9,9 @@ public protocol GDSErrorViewModel {
     var primaryButtonViewModel: ButtonViewModel { get }
     var secondaryButtonViewModel: ButtonViewModel? { get }
 }
+
+/// Conform view models that inherit from ``ErrorViewModel`` to this protocol to set a tertiary button
+@MainActor
+public protocol GDSTertiaryButtonViewModel {
```

Comment:
I'm wondering if this name is quite right, should it be something like:
`GDSScreenWithTertiaryButtonViewModel` ?

Reply:
I am happy with that name, we also have similar protocols for primary and secondary buttons called `PageWithPrimaryButtonViewModel`, `PageWithSecondaryButtonViewModel` and `PageWithTextButtonViewModel`.

I think I should also move this protocol into the same file as these 🤔